IDEAS/Buildings/Components/Interfaces/ZoneInterface.mo

```diff
@@ -160,19 +163,16 @@ equation
     Documentation(revisions="<html>
 <ul>
 <li>
-April 1, 2022, Filip Jorissen:<br/>
```

Comment:
@kldjonge did you remove this change? We need this for openmodelica compatibility.